### PR TITLE
Fixes #11831: wrap text in modal and alerts BZ1205047.

### PR DESCRIPTION
**Pull request by waldenraines**
Ensure text is wrapped in modals and alerts by creating
and using a wordwrap mixin.

http://projects.theforeman.org/issues/11831
https://bugzilla.redhat.com/show_bug.cgi?id=1205047

### CODE REVIEW

File: app/assets/stylesheets/bastion/mixins.less [hunk 1]
```diff
@@ -13,3 +13,15 @@
   -o-box-shadow: @horizontal @vertical @blur @spread @color;
   box-shadow: @horizontal @vertical @blur @spread @color;
 }
+
+.wordwrap
+{
```
Comment (ehelms) [inline app/assets/stylesheets/bastion/mixins.less]: Nitpick - move this to the same line as the declaration

**Comment by waldenraines** [comment]
@ehelms updated.

**Comment by ehelms** [comment]
Did you decide not to go with the mixin in the end?

**Comment by waldenraines** [comment]
> Did you decide not to go with the mixin in the end?

Yeah, because `word-wrap: break-word;` appears to be at least partially supported in all our L1 browsers and because the mixin was causing some weird issues in chrome.  This issue was filed using somewhat of a contrived test case and the odds of someone actually running into this issue are quite low.

**Comment by ehelms** [comment]
ACK

**Pull request merged**